tests/e2e/GetNoticesTest.php

```diff
@@ -2,29 +2,46 @@
 
 namespace Tests\e2e;
 
-use AppBundle\Entity\Notice;
-
 class GetNoticesTest extends BaseApiE2eTestCase
 {
     public function testGetNotice(): void
     {
-        /** @var Notice $validNotice */
-        $validNotice = static::$referenceRepository->getReference('notice_type_ecology');
-        $privateNotice = static::$referenceRepository->getReference('notice_private');
-        $archivedNotice = static::$referenceRepository->getReference('notice_type_ecology_archived');
-        $expiredNotice = static::$referenceRepository->getReference('notice_expired_unpublished');
-        $noticeByDisabledContributor = static::$referenceRepository->getReference('notice_disabled');
-
         $payload = $this->makeApiRequest('/api/v3/notices');
 
+        self::assertCount(8, $payload);
+
         $ids = array_map(static function ($noticeData) {
             return $noticeData['id'];
         }, $payload);
 
-        $this->assertContains($validNotice->getId(), $ids);
-        $this->assertNotContains($privateNotice->getId(), $ids);
-        $this->assertNotContains($archivedNotice->getId(), $ids);
-        $this->assertNotContains($expiredNotice->getId(), $ids);
-        $this->assertNotContains($noticeByDisabledContributor->getId(), $ids);
+        self::assertContains(static::$referenceRepository->getReference('notice_type_ecology')->getId(), $ids);
+        self::assertNotContains(static::$referenceRepository->getReference('notice_private')->getId(), $ids);
+        self::assertNotContains(static::$referenceRepository->getReference('notice_type_ecology_archived')->getId(), $ids);
+        self::assertNotContains(static::$referenceRepository->getReference('notice_expired_unpublished')->getId(), $ids);
+        self::assertNotContains(static::$referenceRepository->getReference('notice_disabled')->getId(), $ids);
+    }
+
+    public function testGetPaginatedNotices(): void
+    {
+        $firstPage = array_map(
+            static function ($noticeData) {
```

Comment:
Juste curieux. Pourquoi `static`?

Reply:
Because I don’t need to access the instance (`$this`) inside the function …

Little heads up from my IDE :
![image](https://user-images.githubusercontent.com/3037833/89904300-08d10380-dbe9-11ea-8285-20eea36280bc.png)